datatypes/src/raster/data_type.rs

```diff
@@ -291,6 +292,51 @@ where
     }
 }
 
+pub trait DefaultNoDataValue {
+    /// The value that is used as default to represent no data for a data type
+    const DEFAULT_NO_DATA_VALUE: Self;
+}
+
+impl DefaultNoDataValue for u8 {
+    const DEFAULT_NO_DATA_VALUE: Self = Self::MAX;
```

Comment:
why not zero?

Reply:
i guess all choices are wrong so i can change it to zero if you like.